CHANGELOG.md

```diff
@@ -32,6 +33,8 @@ BUG FIXES
 * [keys] \#1629 - updating password no longer asks for a new password when the first entered password was incorrect
 * [lcd] importing an account would create a random account
 * [server] 'gaiad init' command family now writes provided name as the moniker in `config.toml`
+* [build] Added Ledger build support via `LEDGER_ENABLED=true|false`
```

Comment:
Do we want this going into `master` or `develop`? Not sure if this is supposed to be part of the diff?

Reply:
I guess into develop now.  I branched this off from master before we switched back to gitflow